### PR TITLE
API Update API to reflect changes to CLI interaction

### PR DESCRIPTION
**Pull request by GuySartorelli**
Reflects changes in https://github.com/silverstripe/silverstripe-framework/pull/11353

## Issue
- https://github.com/silverstripe/silverstripe-framework/issues/11341

### CODE REVIEW

File: src/Task/StaticCacheFullBuildTask.php [hunk 1]
```diff
@@ -3,30 +3,32 @@
...
 use Symbiote\QueuedJobs\Services\QueuedJob;
 use Symbiote\QueuedJobs\Services\QueuedJobService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 
 class StaticCacheFullBuildTask extends BuildTask
 {
-    protected $title = 'Static Cache Full Build';
+    protected static string $commandName = 'static-cache-full-build';
```
Comment (emteknetnz) [inline src/Task/StaticCacheFullBuildTask.php:21]:
```suggestion
    protected static string $commandName = 'StaticCacheFullBuildTask';
```

File: docs/en/building_the_cache.md [hunk 2]
```diff
@@ -4,27 +4,27 @@ There are two main ways the cache is built:
...
 method.
 
 ## Full Site Build
 
 You can generate cache files for the entire site via the `StaticCacheFullBuildJob`.
 This can either be queued up from the QueuedJobs interface or via the task
-`dev/tasks/SilverStripe-StaticPublishQueue-Task-StaticCacheFullBuildTask`. This task also takes a parameter `?startAfter`
-which can delay the execution of the job. The parameter should be in HHMM format, 
-e.g. to start after 2pm, pass `?startAfter=1400`. If it's already after the proposed
+`sake tasks:static-cache-full-build`. This task also takes a parameter `--startAfter`
```
Comment (emteknetnz) [inline docs/en/building_the_cache.md:18]:
```suggestion
`sake tasks:StaticCacheFullBuildTask`. This task also takes a parameter `--startAfter`
```

File: docs/en/building_the_cache.md [hunk 3]
```diff
@@ -4,27 +4,27 @@ There are two main ways the cache is built:
...
 time on the current day, it will push it to the next day.
 
 If you want to do a full site build on a cron, you should do so via the task. An
 example configuration would be:
 
 ```bash
-# build the cache at 1am every day 
-0 1 * * * www-data /path/to/sake dev/tasks/SilverStripe-StaticPublishQueue-Task-StaticCacheFullBuildTask
+# build the cache at 1am every day
+0 1 * * * www-data vendor/bin/sake tasks:static-cache-full-build
```
Comment (emteknetnz) [inline docs/en/building_the_cache.md:28]:
```suggestion
0 1 * * * www-data vendor/bin/sake tasks:StaticCacheFullBuildTask
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: src/Task/StaticCacheFullBuildTask.php (hunk 1 above)
Comment (ChloeSartorelli) [reply src/Task/StaticCacheFullBuildTask.php:21]: Discussion about this and other task command names is being held in the main issue.

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**